api/wasm/v0/cpp/proxy_wasm_externs.h

```diff
@@ -0,0 +1,126 @@
+/*
+ * Proxy-WASM ABI.
+ */
+// NOLINT(namespace-envoy)
+
+#pragma once
+
+#include <stddef.h>
+#include <stdint.h>
+
+//
+//  ABI calls from the host into the VM.
+//
+//  These will typically be implemented by a language specific SDK which will provide an API on top
+//  of this ABI e.g. the C++ SDK provides a proxy_wasm_api.h implementation of the API on top of
+//  this ABI.
+//
+
+// Non-stream calls.
+
+/**
+ * Called when the VM starts for each plugin.
```

Comment:
This is not correct anymore, right? This is called only the first time the root context is created?

Reply:
Fixed.